config/webpack.config.dev.js

```diff
@@ -76,6 +75,25 @@ const getStyleLoaders = (cssOptions, preProcessor) => {
     return loaders;
 };
 
+const getLintingRule = () => {
+    return process.env.LINT_ENV !== 'false' ?
+        {
```

Comment:
this is an anti pattern here - a return that depending on bool value, may return nothing.